### DISCUSSION TITLE
Is WebSocket supported?

### DISCUSSION THREAD

**Discussion by Pradeeps-repo** [Igalia/cog#207] [Q&A] [answered]
I am new to cog. So please bear with me...

I am wondering if cog supports web sockets as out of the box our application with socket communication with a backend nodejs app doesn't work. If it supports, is there a configuration I need to enable? Quick help will be really appreciated.

Thank you

**Comment by dusan-ivanco** [comment, marked as answer]
yes, websocket is forking out of the box
